.github/workflows/check-deployments.yml

```
@@ -0,0 +1,41 @@
+      - main
+      - release/**
+    tags:
+      - "*"
+
+jobs:
+  validate-json:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)